.github/reviewer-lottery.yml

```diff
@@ -0,0 +1,44 @@
+groups:
+  # Default reviewers for all pull requests.
+  # Usually, at least on of the maintainers should approve PR before merging.
+  # The best is if two maintainers do that.
+  - name: maintainers # name of the group
+    reviewers: 2 # how many reviewers do you want to assign?
+    internal_reviewers: 1 # how many reviewers do you want to assign when the PR author belongs to this group?
+    usernames: # github usernames of the reviewers
+      - bmagyar
+      - destogl
+
+  # Reviewers group to get broader feedback.
+  - name: reviewers
+    reviewers: 5
```

Comment:
5 is a little steep, no? 2 should be enough

Reply:
we have some many people. Let's see how this works. They would anyway have a lot things to do. Don't you think?

Reply:
ok